api/adscert.proto

```diff
@@ -58,11 +63,23 @@ message AuthenticatedConnectionVerificationRequest {
 }
 
 // AuthenticatedConnectionVerificationResponse contains the results of verifying
-// signatures.
+// signatures for a request.
 message AuthenticatedConnectionVerificationResponse {
     VerificationStatus verification_status = 1;
-    bool body_valid = 2;
-    bool url_valid = 3;
+    VerificationInfo verification_info = 2;
+}
+
+// AuthenticatedConnectionVerificationBatchRequest contains multiple requests for verifying
+// signatures generated by another party.
+message AuthenticatedConnectionVerificationBatchRequest {
```

Comment:
Are you envisioning that we'll keep both a separate batch verification API and individual verification API, or do you think the batch one will eventually replace the individual API?

Reply:
Up to you. In our previous chat, you mentioned request parameters like waiting for all domain lookups first before processing a batch. Do you think there are settings like this that only apply to batch processing or would you expect them to always be available regardless of the number of requests? 

If there's nothing batch specific then yea we should just merge the APIs so users can batch as needed without changing anything or having different code paths.